types/barnard59-core/index.d.ts

```diff
@@ -7,7 +7,7 @@ import { Logger } from 'winston';
 import LoaderRegistry = require('rdf-loaders-registry');
 export { default as createPipeline } from './lib/factory/pipeline';
 export { default as run } from './lib/run';
-export { Variables } from './lib/Pipeline';
+export { Variables, Context } from './lib/Pipeline';
```

Comment:
Why is this edited?

Reply:
Reverted

Reply:
I re-export some data models from the package root so that they do not have to imported from deep modules in consumer code